src/httpserver/handlers/ok.clj

```diff
@@ -0,0 +1,12 @@
+(ns httpserver.handlers.ok)
+
+(import '(com.td.HttpServer HttpResponse))
+(import '(com.td.HttpServer IHandler))
+
+(deftype ok-handler []
+  com.td.HttpServer.IHandler
+  (generateResponse [this]
+    (doto (HttpResponse.))))
```

Comment:
Also, a forewarning: careful with `doto`. It is designed for stateful objects, which are generally frowned upon in Clojure (but make sense for Java interop stuff). So, if your followup forms don't mutate the initial value `x`, you will just get that original, immutable value back. Contrast this with Clojure's threading macro, `->`, which applies the return value of each function to the next, and returns that final return value:

```clojure
(doto 1 inc inc inc)  ;; 1
(-> 1 inc inc inc)    ;; 4
```